Src/WitsmlExplorer.Frontend/components/Modals/ReportModal.tsx

```diff
@@ -85,9 +86,31 @@ export const ReportModal = (props: ReportModal): React.ReactElement => {
     [report]
   );
 
-  const onCancelButtonClick = () => {
-    JobService.cancelJob(jobId);
+  const cancel = async (jobId: string) => {
+    dispatchOperation({ type: OperationType.HideContextMenu });
     dispatchOperation({ type: OperationType.HideModal });
+    await JobService.cancelJob(jobId);
+    dispatchOperation({ type: OperationType.HideModal });
```

Comment:
Duplicate lines of `dispatchOperation({ type: OperationType.HideModal });`. Probably best to remove the last one?